docs/start/start.md

```diff
@@ -19,11 +14,19 @@ The open-source Amplify Framework provides a set of libraries, UI components, an
 - Amplify UI component libraries for React, React Native, Angular, Ionic and Vue.
 - Amplify Console (accessible in the AWS Console) is an AWS service that provides a git-based workflow for continuous deployment and hosting of fullstack web apps. You can also use the Console to access cloud resources created by the Amplify CLI.
 
-<inline-fragment integration="react" src="~/fragments/react.md"></inline-fragment>
-<inline-fragment integration="react-native" src="~/fragments/reactnative.md"></inline-fragment>
-<inline-fragment integration="android" src="~/fragments/android.md"></inline-fragment>
-<inline-fragment integration="ios" src="~/fragments/ios.md"></inline-fragment>
-<inline-fragment integration="angular" src="~/fragments/web.md"></inline-fragment>
-<inline-fragment integration="ionic" src="~/fragments/web.md"></inline-fragment>
-<inline-fragment integration="vue" src="~/fragments/web.md"></inline-fragment>
-<inline-fragment integration="js" src="~/fragments/web.md"></inline-fragment>
+<inline-fragment integration="ios" src="~/start/getting-started/fragments/ios/build.md"></inline-fragment>
+<inline-fragment integration="ios" src="~/start/getting-started/fragments/android/build.md"></inline-fragment>
```

Comment:
What's up with the `integration` value become `"ios" for everything? If it's no longer a relevant attribute, can we remove it, or rename it to something more neutral?

Reply:
Good catch. Copy paste error. Updated now.